docs/source/install/kernel_install.rst

```diff
@@ -3,37 +3,51 @@
 Installing the IPython kernel
 =============================
 
+.. seealso::
+
+   :ref:`Installing Jupyter <jupyter:install>`
+     The IPython kernel is the Python execution backend for Jupyter.
+
 The Jupyter Notebook and other frontends automatically ensure that the IPython kernel is available.
 However, if you want to use a kernel with a different version of Python, or in a virtualenv or conda environment,
 you'll need to install that manually.
 
-Using the Python version or environment for which you want to set up the kernel, run::
+Kernels for Python 2 and 3
+--------------------------
+
+If you're running Jupyter on Python 3, you can set up a Python 2 kernel like this::
+
+    python2 -m pip install ipykernel
+    python2 -m ipykernel install --user
 
-    pip install ipykernel  # or: conda install ipykernel
+Or using conda, create a Python 2 environment::
+
+    conda create -n ipykernel_py2 python=2 ipykernel
+    source activate ipykernel_py2    # On Windows, remove the word 'source'
     python -m ipykernel install --user
 
+If you're running Jupyter on Python 2 and want to set up a Python 3 kernel,
+follow the same steps, replacing ``2`` with ``3``.
+
 The last command installs a :ref:`kernel spec <jupyterclient:kernelspecs>` file
 for the current python installation. Kernel spec files are JSON files, which
 can be viewed and changed with a normal text editor.
 
-See `python -m ipykernel install --help` for the list of installation options like
-naming the kernel, or non default install location.
-
 .. _multiple_kernel_install:
 
-Multiple IPython installations
-==============================
+Kernels for different environments
+----------------------------------
 
-If you want to have multiple IPython kernels for different environments,
-you will need to specify unique names for the kernelspecs,
-and you may also want to specify the display name of those kernels,
-so that you can clearly see which is which in the notebook menus:
+If you want to have multiple IPython kernels for different virtualenvs or conda environments,
+you will need to specify unique names for the kernelspecs:
 
 .. sourcecode:: bash
 
     source activate myenv
```

Comment:
I would add a comment above here just making it clear that this example (i.e. the `source activate` command) is specific to anaconda. Otherwise this comes off to someone who doesn't know about conda envs that they need to run e.g. `source activate myenv` to install the kernel.